docs/src/refman/transformations.txt

```diff
@@ -202,6 +202,9 @@ If there is no target bitmap, this function returns NULL.
 This is similar to calling `al_invert_transform(al_get_current_transform())`
 but the result of this function is cached.
 
+> *Note*: Allegro's transformation inversion functions work correctly only 
+with 2D transformations (2x2 matrices).
```

Comment:
The 2x2 matrix comment isn't actually right, as 2D transformations are 2x3 as we have the translation there as well. I'd just remove this parenthetical.